src/Controller/Controller.php

```diff
@@ -288,7 +288,7 @@ public function loadComponent(string $name, array $config = []): Component
      * @param string $name Property name
      * @return \Cake\Datasource\RepositoryInterface|null The model instance or null
      */
-    public function __get(string $name)
+    public function __get(string $name): ?\Cake\Datasource\RepositoryInterface
```

Comment:
Doesnt any of the existing sniffers fix the FCQN to use statements? Otherwise I have one here to recommend.